setup.py

```diff
@@ -22,17 +22,30 @@ def write_version_py(filename=None):
         fh.close()
 
 
-write_version_py()
+cfd = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/CFD', 'version.py'
+)
+dem = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/DEM', 'version.py'
+)
+
+write_version_py(cfd)
+write_version_py(dem)
 
 setup(
-    name='simkratos',
+    name='wrappers',
     version=VERSION,
     author='SimPhoNy, EU FP7 Project (Nr. 604005) www.simphony-project.eu',
     description='The Kratos-CFD and Kratos-DEMPack wrappers\
                  for the SimPhoNy framework',
     long_description=README_TEXT,
     entry_points={
-        'simphony.engine': ['kratos = simkratos']},
+        'simphony.engine': [
+            'kratosDEM = wrappers.DEM',
+            'kratosCFD = wrappers.CFD'
+        ]},
     packages=find_packages(),
-    install_requires=["simphony >= 0.1.1"]
+    install_requires=["simphony >= 0.1.5"]
```

Comment:
could you move this back to >= 0.1.3

(0.1.3 is the goal for integration and I suspect this code runs with 0.1.3, 0.1.4 and 0.1.5)